diesel/src/query_builder/delete_statement.rs

```diff
@@ -34,5 +34,10 @@ impl<T, DB> QueryFragment<DB> for DeleteStatement<T> where
         }
         Ok(())
     }
+
+    fn is_safe_to_cache_prepared(&self) -> bool {
+        self.0.from_clause().is_safe_to_cache_prepared() &&
+            self.0.where_clause().map(|w| w.is_safe_to_cache_prepared()).unwrap_or(true)
```

Comment:
This stands out to me as a little odd, I would expect that if we fail to unwrap, we want false, but I suspect I'm just ignorant?